src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -129,7 +129,8 @@ public String runKubectlCommand(String command, ImmutableList<String> args)
   private static String launchAndJoinCommand(Launcher launcher, List<String> args)
       throws IOException, InterruptedException {
     ByteArrayOutputStream cmdLogStream = new ByteArrayOutputStream();
-    int status = launcher.launch().cmds(args).stderr(cmdLogStream).stdout(cmdLogStream).join();
+    int status =
+        launcher.launch().cmds(args).stderr(cmdLogStream).stdout(cmdLogStream).quiet(true).join();
```

Comment:
Would prefer this is gated behind a debug flag so that we can easily enable it.

Reply:
Done.